Branding/WhiteLabel/en.lproj/ForgetMode.strings

```diff
@@ -0,0 +1,40 @@
+/*
+  ForgetMode.strings
+  UserAgent
+
+  Created by Shahen Antonyan on 5/26/20.
+  Copyright © 2020 Cliqz. All rights reserved.
+*/
+
+/* Action label on share extension to immediately open page in a private tab. */
+"ForgetMode.ShareExtension.OpenInPrivateTab" = "Open in New Forget Mode Tab";
+
+/* Title for Automatic forget mode screen */
+"ForgetMode.AutomaticForgetMode.Title" = "Automatic Forget Mode";
```

Comment:
For whitelabel we should probably rename Forget Mode to Private or Incognito Mode